### PR TITLE
Hatching Triage 1.0.1

### PR DESCRIPTION
**Pull request by jbremer**
This PR supersedes #12137.

### CODE REVIEW

**Comment by marcellarichmond** [comment]
@mchasepan

**Comment by moishce** [comment]
@jbremer  - Hi!
Thank you very much for contributing!
Please [sign the CLA](https://cla-assistant.io/demisto/content?pullRequest=12332) so we can continue.

**Comment by moishce** [comment]
@jbremer Please add release notes by runing `demisto-sdk update-release-notes -i Packs/HatchingTriage` 
You can refer to the documentation found here: https://xsoar.pan.dev/docs/integrations/changelog for more information.
or give us write permissions and then I can add them.

**Comment by Brae** [comment]
@moishce I had tried running that command as requested before we pushed the last update but I wasn't getting any output at all. This is all that is produced by that command:

```
demisto-sdk update-release-notes -i Packs/HatchingTriage -u minor
Starting to update release notes.
Either no changes were found in HatchingTriage pack or the changes found should not be documented in the release notes file If relevant changes were made, please commit the changes and rerun the command
```

**Comment by jbremer** [comment]
@moishce FYI we're part of the "NextWave Technology Partner Program" as Hatching.

**Comment by mchasepan** [comment]
> @moishce FYI we're part of the "NextWave Technology Partner Program" as Hatching.

Hey @jbremer Nice to connect again. Yes, we have your TPA on file, but all github contributions must sign our 1pg CLA so we can release into the open source. Please scroll up in this PR and click to sign. Email/Slack w questions. Thx.
image

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py [hunk 1]
```diff
@@ -112,10 +125,53 @@ def get_static_report(client: Client, **args) -> CommandResults:
...
+                        integration_name="Hatching Triage",
+                        score=score
+                    )
+                    indicator = Common.File(
+                        name=r['sample']['target'],
+                        sha256=file['sha256'],
+                        md5=file['md5'],
+                        sha1=file['sha1'],
+                        dbot_score=dbot_score
+                    )
```
Comment (moishce) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py:154]: Please add to the yml file
In commands you changed the new outputs
You can see how to do it [here](https://xsoar.pan.dev/docs/integrations/context-standards-mandatory)

Comment (moishce) [reply Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py:154]: If you want I can help you with that but I need writing permissions.

File: Packs/HatchingTriage/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Hatching Triage
```
Comment (roysagi) [inline Packs/HatchingTriage/README.md:1]:
```suggestion
### Hatching Triage
```

File: Packs/HatchingTriage/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Hatching Triage
+- Added display fields for Integration parameters
```
Comment (roysagi) [inline Packs/HatchingTriage/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Added display fields for the Integration parameters.
```

File: Packs/HatchingTriage/ReleaseNotes/1_0_1.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+- Updated the Docker image to: *demisto/python3:3.9.4.18682*.
+- Added DBot support to static and behavioural report commands
```
Comment (roysagi) [inline Packs/HatchingTriage/ReleaseNotes/1_0_1.md:6]:
```suggestion
- Added DBot support to static and behavioral report commands.
```

File: Packs/HatchingTriage/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+- Fixed error in selecting a Triage profile at submission
```
Comment (roysagi) [inline Packs/HatchingTriage/ReleaseNotes/1_0_1.md:7]:
```suggestion
- Fixed an issue where an error occurred when selecting a Triage profile at submission.
```

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py (hunk 1 above)
Comment (moishce) [reply Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py:154]:
```
- contextPath: File.Name
  description: The full file name (including file extension).
  type: String
- contextPath: File.MD5
  description: The MD5 hash of the file.
  type: String
- contextPath: File.SHA1
  description: The SHA1 hash of the file.
  type: String
- contextPath: File.SHA256
  description: The SHA1 hash of the file.
  type: String
- contextPath: DBotScore.Indicator
  description: The indicator that was tested.
  type: String
- contextPath: DBotScore.Type
  description: The indicator type.
  type: String
- contextPath: DBotScore.Vendor
  description: The vendor used to calculate the score.
  type: String
- contextPath: DBotScore.Score
  description: The actual score.
  type: Number
- contextPath: URL.Data
  description: The URL
  type: String
```
Need to add it on the yml file in the outputs of the commands:
"**Triage-get-static-report**" in line 278
"**Triage-get-report-triage**" in line 299

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml [hunk 6]
```diff
@@ -275,6 +278,18 @@ script:
...
+        type: String
+      - contextPath: DBotScore.Type
+        description: The indicator type - File or URL
+        type: String
+      - contextPath: DBotScore.Vendor
+        description: The integration used to generate the indicator
+        type: String
+      - contextPath: DBotScore.Score
+        description: Analysis verdict as score from 1 to 10
+        type: Number
```
Comment (moishce) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml:292]:
```suggestion
        type: Number
      - contextPath: File.Name
        description: The full file name (including file extension).
        type: String
      - contextPath: File.MD5
        description: The MD5 hash of the file.
        type: String
      - contextPath: File.SHA1
        description: The SHA1 hash of the file.
        type: String
      - contextPath: File.SHA256
        description: The SHA1 hash of the file.
        type: String
      - contextPath: URL.Data
        description: The URL
        type: String
```

File: Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml [hunk 7]
```diff
@@ -296,6 +311,18 @@ script:
...
```
Comment (moishce) [inline Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml:325]:
```suggestion
        type: Number
      - contextPath: File.Name
        description: The full file name (including file extension).
        type: String
      - contextPath: File.MD5
        description: The MD5 hash of the file.
        type: String
      - contextPath: File.SHA1
        description: The SHA1 hash of the file.
        type: String
      - contextPath: File.SHA256
        description: The SHA1 hash of the file.
        type: String
      - contextPath: URL.Data
        description: The URL
        type: String
```

**Comment by ShahafBenYakir** [comment]
Hi @jbremer, we need to move forward with this pull request, please address the relevant comments. If there will be no progress we will need to close this PR.

**Review by moishce — APPROVED** [review]

**Pull request merged**